spec/controllers/enrollments_controller_spec.rb

```diff
@@ -51,6 +51,54 @@
   end
 
   describe 'GET #index' do
+    describe "I have dgfip api_particulier and api_entreprise enrollments" do
+      let(:dgfip_enrollments) { FactoryGirl.create_list(:enrollment, 3, fournisseur_de_donnees: 'dgfip') }
+      let(:api_particulier_enrollments) { FactoryGirl.create_list(:enrollment, 4, fournisseur_de_donnees: 'api-particulier') }
+      let(:api_entreprise_enrollments) { FactoryGirl.create_list(:enrollment, 5, fournisseur_de_donnees: 'api-entreprise') }
+
+      describe "I have a dgfip user" do
+        let(:user) { FactoryGirl.create(:user, uid: uid, provider: 'dgfip', email: 'test@test.test') }
+
+        it 'returns a success response' do
+          dgfip_enrollments
+          api_particulier_enrollments
+          api_entreprise_enrollments
+          get :index
+
+          json = JSON.parse(response.body)
+          expect(json.count).to eq(3)
+        end
+      end
+
+      describe "I have a api_particulier user" do
+        let(:user) { FactoryGirl.create(:user, uid: uid, provider: 'api_particulier', email: 'test@test.test') }
+
+        it 'returns a success response' do
+          dgfip_enrollments
+          api_particulier_enrollments
+          api_entreprise_enrollments
+          get :index
+
+          json = JSON.parse(response.body)
+          expect(json.count).to eq(4)
+        end
+      end
+
+      describe "I have a api_entreprise user" do
+        let(:user) { FactoryGirl.create(:user, uid: uid, provider: 'api_entreprise', email: 'test@test.test') }
+
+        it 'returns a success response' do
```

Comment:
J'aurais changé la description du test en quelque chose comme `it 'returns the right number of enrollments'`